asmcomp/arm64/emit.mlp

```diff
@@ -711,12 +717,26 @@ let emit_instr env i =
           `	b	{emit_label env.f.fun_tailrec_entry_point_label}\n`
         else
           output_epilogue env (fun () -> `	b	{emit_symbol func}\n`)
-    | Lop(Iextcall { func; alloc = false; }) ->
-        `	bl	{emit_symbol func}\n`
-    | Lop(Iextcall { func; alloc = true; }) ->
-        emit_load_symbol_addr reg_x8 func;
-        `	bl	{emit_symbol "caml_c_call"}\n`;
-        `{record_frame env i.live (Dbg_other i.dbg)}\n`
+    | Lop(Iextcall {func; alloc; stack_ofs}) ->
+        if stack_ofs > 0 then begin
+          ` mov {emit_reg reg_stack_arg_begin}, sp\n`;
+          ` add {emit_reg reg_stack_arg_end}, sp, #{emit_int (Misc.align stack_ofs 16)}\n`;
+          emit_load_symbol_addr reg_x8 func;
+          ` bl  {emit_symbol "caml_c_call_stack_args"}\n`;
+          `{record_frame env i.live (Dbg_other i.dbg)}\n`
+        end else if alloc then begin
+          emit_load_symbol_addr reg_x8 func;
+          ` bl  {emit_symbol "caml_c_call"}\n`;
+          `{record_frame env i.live (Dbg_other i.dbg)}\n`
+        end else begin
+          ` mov {emit_reg reg_x19}, sp\n`;
```

Comment:
For readability, add a comment that it is callee-saved.

Reply:
Done!

Reply:
Thanks.